README.md

```diff
@@ -27,6 +27,10 @@ More details of the steps included in this project, and running instructions, ca
 3. [skills_extraction](skills_taxonomy_v2/pipeline/skills_extraction/README.md) - Extracting skills from skill sentences.
 4. [skills_taxonomy](skills_taxonomy_v2/pipeline/skills_taxonomy/README.md) - Building the skills taxonomy from extracted skills.
 
+### Extract skills example
+
+A simple example of extracting skills from a toy dataset of job adverts is given in [the examples folder](skills_taxonomy_v2/examples/extract_skills.py).
```

Comment:
Nice!! really cool - I'd rename the files so that it's clear which file does what - looks like extract_skills.py has the functions needed to run Extract Skills - something like extract_skills_utils.py?